utility/src/test/java/org/oppia/android/util/logging/EventBundleCreatorTest.kt

```diff
@@ -139,6 +145,12 @@ class EventBundleCreatorTest {
   @Inject lateinit var context: Context
   @Inject lateinit var eventBundleCreator: EventBundleCreator
 
+  @Parameter
+  lateinit var screenNameCase: String
+
+  @Parameter
+  lateinit var screenNameCaseValue: String
```

Comment:
```suggestion
  @Parameter lateinit var name: String
  @Parameter lateinit var expNameStr: String

  private val screenName by lazy { ScreenName.valueOf(name) }
```

Kotlin actually has a built-in way to convert strings to enums, so long as the name constant perfectly matches. Also, the shorter parameter names can help cut down on how long the iterations are.

Reply:
done.